### DISCUSSION TITLE
offline install can in other directory ?

### DISCUSSION THREAD

**Discussion by yezhengmao1** [netdata/netdata#13311] [Q&A] [answered]
I saw the *.run script will install netdata in /opt/netdata directory and it`s static? But my /opt directory is set to read-only.
Can install it in other directories?

**Comment by thiagoftsm** [comment, marked as answer]
Hello @yezhem ,

It looks like we still do not have full support for this, I tested this morning and I wrote this [issue](https://github.com/netdata/netdata/issues/13321) to fix what is necessary. @Ferroin am I missing something here?

Best regards!

**Reply by Ferroin** [reply]
We’ve never supported this, and that’s unlikely to change. The agent itself hard-codes a large number of paths internally, so even though we can kind of put the files in a different place, a lot of things won’t actually work correctly at runtime.
